providers/edgeexecutor/docs/commits.rst

```diff
@@ -29,135 +29,19 @@ Package apache-airflow-providers-edgeexecutor
 Handle edge workers on remote sites via HTTP(s) connection and orchestrates work over distributed sites
 
 
-This is detailed commit list of changes for versions provider package: ``edge``.
+This is detailed commit list of changes for versions provider package: ``edgeexecutor``.
 For high-level changelog, see :doc:`package information including changelog <index>`.
 
 
 
 1.0.0
 .....
 
-Latest change: 2025-04-05
+Latest change: 2025-04-15
 
-==================================================================================================  ===========  ============================================================================================================================================================
+==================================================================================================  ===========  ===================================================================================================
 Commit                                                                                              Committed    Subject
-==================================================================================================  ===========  ============================================================================================================================================================
```

Comment:
Are we losing history here? Is it cos we now release 1.0 of "edgeexecutor" and not edge?

Reply:
Probably, but this is automated file so we can always look into it later

Reply:
cc @potiuk

Reply:
Probably. But it also does not matter too much as it has never been released officially so what's "before" the first release does not really matter for commit changelog, but yes - we can fix it later.